api/apikeys.go

```diff
@@ -3,16 +3,91 @@ package api
 import (
 	"net/http"
 	"strconv"
+	"time"
 
 	"github.com/SkynetLabs/skynet-accounts/database"
 	"github.com/julienschmidt/httprouter"
 	"gitlab.com/NebulousLabs/errors"
+	"go.mongodb.org/mongo-driver/bson/primitive"
 	"go.mongodb.org/mongo-driver/mongo"
 )
 
+type (
+	// APIKeyPOST describes the body of a POST request that creates an API key
+	APIKeyPOST struct {
+		Public   bool     `json:"public,string"`
+		Skylinks []string `json:"skylinks"`
+	}
+	// APIKeyPUT describes the request body for updating an API key
+	APIKeyPUT struct {
+		Skylinks []string
```

Comment:
no `json` annotation here?

Reply:
No need - we are not doing anything custom, so the standard `skylinks` representation is exactly what we want.